content/docs/clients/javascript/angular-example.md

```diff
@@ -1,38 +0,0 @@
----
```

Comment:
I verified that this works for those wanting to integrate with angularjs 1.x, we don't need a sample app for every integration. I'm fine with removing this though as people would be integrating with newer versions of angular.

Reply:
Cool, that makes sense. If you ever want me to throw it back in, I can grab it from source control.